packages/icons/svgr.config.js

```diff
@@ -20,13 +21,18 @@ export const ${componentName} = ({
   ${jsx}
 )
 
+const ${componentName} = styled(BaseComponent)\`
+  outline: none;
+\`
```

Comment:
```suggestion
`
```

Reply:
From my older PR: Steve's comment on this 
it broke on nested backticks, this was the only thing that worked - since we are inside the template backticks that start on line 9 https://github.com/pricelinelabs/design-system/blob/Fix-outline-property-for-icons/packages/icons/svgr.config.js#L9 - we want them rendered into the built template, not evaluated.

Icon uses these svgr-built components, so it should be there - https://github.com/pricelinelabs/design-system/blob/Fix-outline-property-for-icons/packages/icons/src/Icon.js#L8